packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2925,6 +2934,11 @@ export default class Meeting extends StatelessWebexPlugin {
       return Promise.reject(new Error(errorMessage));
     }
 
+    // If level is already the same, don't do anything
```

Comment:
Is this necessary? what's the cost of reassigning the quality level?

Reply:
Everything after this if block will cause changes to the active stream
Why mess with the stream if we aren't gonna actually change anything

```js
	// If we are receiving media, set the remote media quality
    if (receiveAudio || receiveVideo) {
      qualityActions.push(this.setRemoteQualityLevel(level));
    }
    // If we are sending media, set the local media quality
    if (sendVideo) {
      qualityActions.push(this.setLocalVideoQuality(level));
    }
```

Reply:
I like this change, but would also like to see it in the `setRemoteQualityLevel` and `setLocalVideoQuality` commands so they don't call `updateMedia` extraneously if called directly.

Reply:
updated and added